analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces-expected-output.yml

```diff
@@ -19,7 +19,13 @@ project:
   homepage_url: ""
   scopes:
   - name: "dependencies"
-    dependencies: []
+    dependencies:
```

Comment:
Commit message nits:
"one want**s** to see"
"Yarn create**s** one"
"and **use** them for"

Reply:
Done.